lib/Cake/Test/Case/View/JsonViewTest.php

```diff
@@ -28,12 +28,27 @@
  */
 class JsonViewTest extends CakeTestCase {
 
+/**
+ * setUp method
+ *
+ * @return void
+ **/
 	public function setUp() {
 		parent::setUp();
 		Configure::write('debug', 0);
 	}
 
 /**
+ * tearDown method
+ *
+ * @return void
+ **/
+	public function tearDown() {
+		parent::tearDown();
+		restore_error_handler();
```

Comment:
You should only restore the error handler when you change it. Otherwise you can end up popping off phpunits error handler.